misk/src/test/kotlin/misk/web/proxy/WebProxyActionTest.kt

```diff
@@ -262,6 +262,32 @@ class WebProxyActionTest {
         .build()
   }
 
+  @Test
+  internal fun getForwardedLongPathMatchTrailingSlash() {
+    upstreamServer.enqueue(MockResponse()
+        .setResponseCode(418)
+        .addHeader("UpstreamHeader", "UpstreamHeaderValue")
+        .setBody("I am an intercepted response!"))
+
+    val responseAsync = async {
+      httpClient.newCall(
+          get("/local/prefix/tacos/another/long/prefix/see/if/forwards/", weirdMediaType)).execute().toMisk()
```

Comment:
where does weirdMediaType come from?

Reply:
Pulled it from NotFoundActionTest and ContentBasedDispatchTest to ensure that requests with  funky MediaTypes are still forwarded to upstream servers.